src/main/java/com/softserve/academy/spaced/repetition/controller/CourseController.java

```diff
@@ -98,7 +97,7 @@
     @PreAuthorize(value = "@accessToUrlService.hasAccessToCategory(#category_id)")
     public ResponseEntity<CoursePublicDTO> addCourse(@RequestBody Course course, @PathVariable Long category_id) {
         courseService.addCourse(course, category_id);
```

Comment:
We should get object of `Course` from method `addCourse` and use it in DTOBuilder (not sure)